doc/modules/model_evaluation.rst

```diff
@@ -96,17 +96,32 @@ Scoring                                Function
 'neg_mean_squared_error'               :func:`metrics.mean_squared_error`
 'neg_root_mean_squared_error'          :func:`metrics.root_mean_squared_error`
 'neg_mean_squared_log_error'           :func:`metrics.mean_squared_log_error`
-'neg_root_mean_squared_log_error'      :func:`metrics.root_mean_squared_log_error`
```

Comment:
What is the reason for removing this metric. It looks like this is available:

```python
>>> from sklearn.metrics import get_scorer
>>> get_scorer("neg_root_mean_squared_log_error")
make_scorer(root_mean_squared_log_error, greater_is_better=False, response_method='predict')
```

But I'm might have missed something.

Reply:
Thank you for spotting this — I'd run `metrics.get_scorer_names()` in the environment with an old sklearn version. Will update back!

Reply:
Indeed, we might have added last release ;)